rosinstall_noetic_core20/snapped_ros1_pkg.rosinstall

```diff
@@ -0,0 +1 @@
+- git: {local-name: snapped_ros1_pkg, uri: 'https://github.com/Guillaumebeuzeboc/snapped_ros1_pkg.git', version: main}
```

Comment:
Use the `ubuntu-robotics` repo.

Reply:
This is an issue, because the https://github.com/ubuntu-robotics/ros-snaps-examples.git repository contains multiple package and most importantly, multiple packages with the same name. We should first rename the all the ros packages with a unique name and then select only the package we want with `--packages-select`

Reply:
Oh I see. How about using one of the upstream demo package instead of this then?